packages/react-native-bpk-component-calendar/src/android/src/main/java/net/skyscanner/backpack/reactnative/calendar/CalendarPackage.kt

```diff
@@ -21,14 +21,15 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.bridge.NativeModule
 import com.facebook.react.bridge.ReactApplicationContext
 import com.facebook.react.uimanager.ViewManager
+import javax.annotation.Nonnull
 
 class CalendarPackage : ReactPackage {
 
-  override fun createNativeModules(reactContext: ReactApplicationContext?): MutableList<NativeModule> {
+  override fun createNativeModules(reactContext: ReactApplicationContext): MutableList<NativeModule> {
     return mutableListOf()
   }
 
-  override fun createViewManagers(reactContext: ReactApplicationContext?): MutableList<ViewManager<*, *>> {
+  override fun createViewManagers(reactContext: ReactApplicationContext): MutableList<ViewManager<*, *>> {
```

Comment:
After upgrading Gradly things I started seeing build errors here that we were failing to override the methods we were supposed to. I checked and it appears the signatures changed so that `ReactApplicationContext` is no longer optional. This change doesn't appear to break anything.